extensions/amp-next-page/0.1/test/validator-amp-next-page.out

```diff
@@ -18,7 +18,7 @@ PASS
 |  </head>
 |  <body>
 |    <h2>Content discovery</h2>
-|    <amp-next-page>
+|    <amp-next-page src="https://foo.com/data.json">
```

Comment:
would be better to have two tests: 1- only has `src`, 2- only has inline `script` config. we probably want to disallow both but not sure if possible to do "one-of" `src` or `child <script>` @honeybadgerdontcare  is it?

Reply:
That is currently not possible in the validator engine as a one-of specifically between a child element and an attribute but it could be done as two different tag specs. One would require the json script without a src attribute, the other wouldn't require the json script but would have a mandatory src attribute.

Reply:
Done

Reply:
This is still only one test. We would want 
1- one amp-next-page with `src` not inline script
2- one amp-next-page with inline script, not `src`

Reply:
I added a separate `amp-next-page-inline.html` file for the second test as you can only have one `amp-next-page` per document.

Reply:
Makes sense. Thanks.